style-guide/readme.md

```diff
@@ -0,0 +1,51 @@
+> **Goal of this guide:** to help you write to fit a specific context, purpose, or audience.
+
+# How to refine your writing style
+
+You already have a writing style. It's what sets your writing apart and makes it unique. We're sure it's ridiculously awesome. But we're always looking to improve and to do everything more effectively, therefore we want to think about why awesome writing is awesome so that we can apply those reasons to everything we write.
+
+## Recognise what style is and what it's building blocks are
+
+The internet is already flooded with style guides, so no need to reinvent the wheel here. Check out:
+* [this guide](http://www.learnnc.org/lp/editions/few/684), which focuses on word choice, sentence fluency and voice,
+* [or this one](http://literarydevices.net/style/), which is about different types of style,
+* [or perhaps this one](http://www.writersdigest.com/writing-articles/by-writing-goal/improve-my-writing/brush-up-on-your-style-in-10-minutes-or-less), which is a practical list of suggestions to improve style.
+
+You get the idea.
+
+## Ask yourself questions as you write
+
+As with everything, asking questions is important during the process writing and determining style.
+
+* For whom are you writing (**audience**)?
+* What is the **context**?
+* Why do they need to know what you are going to tell them (**purpose**)?
+
+Style is not a matter of right and wrong but of what is appropriate for a particular setting and audience. To state the obvious (hey, sometimes it needs to be done), if you're writing about quicksand, your style is going to be vastly different if you are writing for children as opposed to your fellow scientific quicksand researchers.
+
+## Read stuff and pay attention
+
+When you read something you like, think about what's great about it and why. How did the author connect to you? We will demonstrate.
+
+[This](http://waitbutwhy.com/2015/01/artificial-intelligence-revolution-1.html) is an extremely long but very well-written blog post about artificial intelligence. You only need to read the first couple of paragraphs to get a good idea of the writer's style (otherwise you'll still be reading when we're all having our first coffee tomorrow morning).
+
+Here are some following thoughts:
+
+I remain engaged despite it's crazy length. Why?
```

Comment:
I liked that you made a list out of the questions you had. It causes a good overview.